<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDatabaseCleanup</class>
 <widget class="QDialog" name="FormDatabaseCleanup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cleanup database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="m_grpCleanupSettings">
     <property name="title">
      <string>Cleanup settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0" colspan="3">
       <widget class="QCheckBox" name="m_checkShrink">
        <property name="text">
         <string>Optimize and fix database file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="m_checkRemoveReadMessages">
        <property name="text">
         <string>Remove all read articles</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="m_spinDays">
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="m_checkRemoveRecycleBin">
        <property name="text">
         <string>Remove all articles from recycle bin</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="m_checkRemoveOldMessages">
        <property name="text">
         <string>Remove all articles older than</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QCheckBox" name="m_checkRemoveStarredMessages">
        <property name="text">
         <string>Remove all starred articles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Database information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_lblFileSize">
        <property name="text">
         <string>Total data size</string>
        </property>
        <property name="buddy">
         <cstring>m_txtFileSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_txtFileSize">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_lblDatabaseType">
        <property name="text">
         <string>Database type</string>
        </property>
        <property name="buddy">
         <cstring>m_txtDatabaseType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_txtDatabaseType">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="LabelWithStatus" name="m_lblResult" native="true">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="m_progressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_btnBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_checkRemoveReadMessages</tabstop>
  <tabstop>m_checkRemoveRecycleBin</tabstop>
  <tabstop>m_checkShrink</tabstop>
  <tabstop>m_checkRemoveOldMessages</tabstop>
  <tabstop>m_spinDays</tabstop>
  <tabstop>m_txtFileSize</tabstop>
  <tabstop>m_txtDatabaseType</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_btnBox</sender>
   <signal>rejected()</signal>
   <receiver>FormDatabaseCleanup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkRemoveOldMessages</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_spinDays</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
